llvm/test/CodeGen/WebAssembly/unreachable.ll

```diff
@@ -1,33 +1,130 @@
-; RUN: llc < %s -asm-verbose=false -verify-machineinstrs | FileCheck %s
-; RUN: llc < %s -asm-verbose=false -fast-isel -fast-isel-abort=1 -verify-machineinstrs | FileCheck %s
-
-; Test that LLVM unreachable instruction and trap intrinsic are lowered to
-; wasm unreachable
+; RUN: llc < %s -verify-machineinstrs | FileCheck %s
```

Comment:
I'm not sure if I understand what all the new tests in this file are testing. Currently this file emits the same results for all combination of `--trap-unreachable` and `--no-trap-after-noreturn`. Also without this PR the results are the same. Not sure what this PR is trying to achieve...

Reply:
Check out the first commit: [Add no-trap-after-noreturn flag and wasm tests](https://github.com/llvm/llvm-project/pull/65876/commits/7c43c803764bf9e0256d4e3e9f497d2622bb8f69). The file definitely didn't emit the same results then! Remember, the --no-trap-after-noreturn option didn't exist before this PR :)